docs/migrate.rst

```diff
@@ -1,6 +1,15 @@
 Migration Guide
 ===============
 
+Migrating to Rally 1.2.0
+------------------------
+
+CPU usage is not measured anymore
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+With Rally 1.2.0, CPU usage will neither be measured nor reported. We suggest to use system monitoring tools like ``vmstat`` or `Metricbeat <https://www.elastic.co/downloads/beats/metricbeat>`_ to measure CPU usage instead.
```

Comment:
nit: I'd rather mention `mpstat` or `sar` instead of `vmstat` for CPU. Taking a quick glance of where tot CPU time gets spent with `vmstat` is fine too of course.

Reply:
addressed in https://github.com/elastic/rally/pull/696/commits/59fa4fbc8c910447734ac4ff47350916f483051a